website/docs/r/container_cluster.html.markdown

```diff
@@ -202,6 +207,8 @@ The `ip_allocation_policy` block supports:
     ClusterIPs. This must be an existing secondary range associated with the cluster
     subnetwork.
 
+> Please note that this activate Ip aliases. See the [official documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/ip-aliases)
```

Comment:
I think a better place for this would be in the section on `ip_allocation_policy` at line 102. Then you wouldn't need for it to be a separate note, but just part of the description.

(also nit: activates, IP)

Reply:
I moved it to line 102 as you suggested
I changed the wording slightly to fit w/h what was used line 96 (I'm not satisfied with it so if you have any suggestion)